index.html

```diff
@@ -46,7 +46,7 @@
 					<div class="social-media module">
 						<h1>Follow us</h1>
 						<div class="follow-nature">
-							<ul>
+							<ul class="cleared">
```

Comment:
ninja skills Ben. Thanks for the pointers here. The inspector tool in chrome is very powerful. I'm still trying to understand why they have floated the #extranav to the right.